e2e/tests/chat-flow.spec.ts

```
@@ -15,119 +15,110 @@ function urlEndsWith(suffix: string): (url: URL) => boolean {
+
+    // Navigate to session B via the URL directly (same entries fixture).
+    await page.goto(`/chat/${SESSION_B.id}`);
+    await expect(page.locator("text=Hi there!").first()).toBeVisible({
+      timeout: 5_000,
+    });
   });
 
-  test("prefers server-side AI title over first-user-message preview", async ({
+  test("history panel shows session previews", async ({
     page,
   }) => {
```

ESLint

[error] Replace `⏎····page,⏎·` with `·page`

(prettier/prettier)